travis/setup-php-extensions.sh

```diff
@@ -49,10 +49,13 @@ echo 'opcache.enable=0' >> "$php_ini_file"
 # APCu
 if [ `php -r "echo (int) (version_compare(PHP_VERSION, '7.0.0-dev', '>=') && version_compare(PHP_VERSION, '7.3.0-dev', '<'));"` == "1" ]
```

Comment:
@marc1706 Why between this two versions? Can this condition be replaced by the new one?

Reply:
That's what was in place, which version did you have in mind again?

Reply:
Perhaps starting from 7.1, not sure though.

Reply:
The APCu extension was not available via pecl in 7.3 yet. Not sure if this has changed yet.

Reply:
Then the check should be check if is available or something and is not installed